src/components/Board.vue

```diff
@@ -1,89 +1,129 @@
-    <div id="touch-area">
-      <div id="good">
-        <div id="excellent">
-        </div>
+    <button class="button-start" style="margin-top: 80px" @click="stopGame">stop</button>
+    <div id="good" ref="good">
+      <div id="excellent" ref="excellent">
       </div>
     </div>
+    <arrow
+      v-for="n in storeItems"
+      :key="n.id"
+    >
+    </arrow>
   </div>
 </template>
 
 <script lang="ts">
 
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
-import { Block } from '@/types'
-
-@Component
+import Arrow from '@/gameComponents/Arrow.vue'
+import { EDirection, IFlowProps } from '@/types'
+import { StoreModuleEnum } from '@/store/types'
+import { EActionScore } from '@/store/modules/score/typesScore'
+import { EActionArrow, IArrowData } from '@/store/modules/arrow/typesArrow'
+
+@Component({
+  components: {
+    Arrow
+  }
+})
 export default class Board extends Vue {
-  gameFlow = document.getElementById('game')
-  scoreElement = document.getElementById('score')
-  scoreCounter: number = 0
-  gameFlowHeight = 600
-  block = new Block()
-  counts: number = 0
-
-  checkTouch (item: any) {
-    const itemPosition = parseInt(window.getComputedStyle(item).getPropertyValue('top'))
-    const grMinHeight = 90
-    const grMaxHeight = 60
-    const goodMinHeight = 110
-    const goodMaxHeight = 30
-    const greatArea = itemPosition >= this.gameFlowHeight - grMinHeight && itemPosition <= this.gameFlowHeight - grMaxHeight
-    const goodArea = itemPosition >= this.gameFlowHeight - goodMinHeight && itemPosition <= this.gameFlowHeight - goodMaxHeight
-
-    if (greatArea) {
-      this.scoreCounter += 2
-    } else if (goodArea) {
-      this.scoreCounter += 1
+  isPlay: boolean = false
+  safeLoop: number = 0
+
+  get storeItems (): IArrowData[] {
+    return this.$store.state.arrowStore.arrowsData
+  }
+
+  get positions (): IFlowProps {
+    const goodArea: any = this.$refs.good
+    const exArea: any = this.$refs.excellent
+    return {
+      goodArTop: goodArea.getBoundingClientRect().top,
+      goodArBottom: goodArea.getBoundingClientRect().bottom,
+      exAreaTop: exArea.getBoundingClientRect().top,
+      exAreaBottom: exArea.getBoundingClientRect().bottom
     }
-  this.scoreElement!.textContent = this.scoreCounter.toString()
   }
 
-  delay = (ms: number) => {
-    return new Promise(resolve => setTimeout(resolve, ms))
+  startGame (): void {
+    console.log('the game is on')
+    this.isPlay = true
+    this.runRender()
+    window.addEventListener('keydown', this.logKey)
   }
 
-  renderBlocks = async (elements: any) => { // function will be deleted
-    for (let i = 0; i < elements.length; i += 1) {
-      if (i === 0) {
-        this.gameFlow?.appendChild(elements[i]) // components will be added using VUE methods
+  runRender (): void {
+    const interval = setInterval(() => {
+      this.$store.dispatch(`${StoreModuleEnum.arrowStore}/${EActionArrow.ADD_DATA}`)
```

Comment:
could you extract this line into a method with a meaningful name? If think it will be easy to understand what it does